src/deadline/client/cli/_groups/bundle_group.py

```diff
@@ -98,9 +98,14 @@ def validate_parameters(ctx, param, value):
     type=click.Choice([e.value for e in JobAttachmentsFileSystem]),
 )
 @click.option(
-    "--yes",
+    "--upload",
     is_flag=True,
-    help="Skip any confirmation prompts",
+    help="Indicates that files will be uploaded to S3 if needed",
+)
+@click.option(
```

Comment:
@mwiebe explicitly said we shouldn't just have a --trust option but to add an allow list of paths.

Thoughts ?

Reply:
Keeping the `--yes` option to mean accept default prompts seems like a good idea to me. What's our rationale for removing it?

Reply:
It doesn't behave that way I or customers expect. Removal may be part of a bigger change than in this one, but this change does propose changed behaviour by switching to `--upload` (something like that).

I do think we should have an option shared amongst all (or most, can't think of exception atm) options that indicate you're operating in an session that is not interactive and thus, anything that would prompt or is not answered would cause a failure of the command. `--yes` is currently the way to get that, but that's not great